parlai/mturk/tasks/acute_eval/README.md

```diff
@@ -0,0 +1,108 @@
+# ACUTE-Eval
+
+## Paper information
+
+Margaret Li, Jason Weston, Stephen Roller.
+_[ACUTE-EVAL: Improved Dialogue Evaluation with Optimized Questions and Multi-turn Comparisons](https://arxiv.org/abs/1909.03087)_.
+
+## Citation
+
+If you use this evaluation method in your own work, please cite with the
+following BibTex entry:
+
+  @misc{li2019acuteeval,
+    title={ACUTE-EVAL: Improved Dialogue Evaluation with Optimized Questions and Multi-turn Comparisons},
```

Comment:
Why not the workshop?

Reply:
It's not archival? Does it still have proceedings that can be cited?